assets/xml/objects/object_zl1.xml

```diff
@@ -1,90 +1,119 @@
-        <Limb Name="object_zl1_Standardlimb_00F4EC" Type="Standard" EnumName="OBJECT_ZL1_LIMB_04" Offset="0xF4EC" />
-        <Limb Name="object_zl1_Standardlimb_00F4F8" Type="Standard" EnumName="OBJECT_ZL1_LIMB_05" Offset="0xF4F8" />
-        <Limb Name="object_zl1_Standardlimb_00F504" Type="Standard" EnumName="OBJECT_ZL1_LIMB_06" Offset="0xF504" />
-        <Limb Name="object_zl1_Standardlimb_00F510" Type="Standard" EnumName="OBJECT_ZL1_LIMB_07" Offset="0xF510" />
-        <Limb Name="object_zl1_Standardlimb_00F51C" Type="Standard" EnumName="OBJECT_ZL1_LIMB_08" Offset="0xF51C" />
-        <Limb Name="object_zl1_Standardlimb_00F528" Type="Standard" EnumName="OBJECT_ZL1_LIMB_09" Offset="0xF528" />
-        <Limb Name="object_zl1_Standardlimb_00F534" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0A" Offset="0xF534" />
-        <Limb Name="object_zl1_Standardlimb_00F540" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0B" Offset="0xF540" />
-        <Limb Name="object_zl1_Standardlimb_00F54C" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0C" Offset="0xF54C" />
-        <Limb Name="object_zl1_Standardlimb_00F558" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0D" Offset="0xF558" />
-        <Limb Name="object_zl1_Standardlimb_00F564" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0E" Offset="0xF564" />
-        <Limb Name="object_zl1_Standardlimb_00F570" Type="Standard" EnumName="OBJECT_ZL1_LIMB_0F" Offset="0xF570" />
-        <Limb Name="object_zl1_Standardlimb_00F57C" Type="Standard" EnumName="OBJECT_ZL1_LIMB_10" Offset="0xF57C" />
-        <Limb Name="object_zl1_Standardlimb_00F588" Type="Standard" EnumName="OBJECT_ZL1_LIMB_11" Offset="0xF588" />
-        <Skeleton Name="object_zl1_Skel_00F5D8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_ZL1_LIMB_NONE" LimbMax="OBJECT_ZL1_LIMB_MAX" EnumName="ObjectZl1Limb" Offset="0xF5D8" />
-        <Animation Name="object_zl1_Anim_010B38" Offset="0x10B38" />
-        <Animation Name="object_zl1_Anim_011348" Offset="0x11348" />
-        <Animation Name="object_zl1_Anim_0116E4" Offset="0x116E4" />
-        <Animation Name="object_zl1_Anim_011B88" Offset="0x11B88" />
-        <Animation Name="object_zl1_Anim_012118" Offset="0x12118" />
-        <Animation Name="object_zl1_Anim_012B04" Offset="0x12B04" />
-        <Animation Name="object_zl1_Anim_012F80" Offset="0x12F80" />
-        <Animation Name="object_zl1_Anim_0132D8" Offset="0x132D8" />
-        <Animation Name="object_zl1_Anim_0138E0" Offset="0x138E0" />
-        <Animation Name="object_zl1_Anim_013F10" Offset="0x13F10" />
-        <Animation Name="object_zl1_Anim_0143A8" Offset="0x143A8" />
+        <!-- Palette -->
+        <Texture Name="gChildZelda1TLUT_0450" OutName="child_zelda_1_tlut_0450" Format="rgba16" Width="16" Height="16" Offset="0x0450" />
+
+        <!-- Textures -->
+        <Texture Name="gChildZelda1DressGradientTex" OutName="child_zelda_1_dress_gradient" Format="ci8" Width="8" Height="8" Offset="0x0650" TlutOffset="0x0450" />
```

Comment:
```suggestion
        <Texture Name="gChildZelda1TLUT_0450" OutName="child_zelda_1_tlut_0450" Format="rgba16" Width="16" Height="16" Offset="0x450" />

        <!-- Textures -->
        <Texture Name="gChildZelda1DressGradientTex" OutName="child_zelda_1_dress_gradient" Format="ci8" Width="8" Height="8" Offset="0x650" TlutOffset="0x450" />
```
We've put effort to remove leading zero's, could you double check leading 0's in the offsets/tlutOffsets?